server.go

```diff
@@ -1587,14 +1589,28 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 // utilize the ordering of the local and remote public key. If we didn't use
 // such a tie breaker, then we risk _both_ connections erroneously being
 // dropped.
-func shouldDropLocalConnection(local, remote *btcec.PublicKey) bool {
+func (s *server) shouldDropLocalConnection(local, remote *btcec.PublicKey) bool {
```

Comment:
Does this no longer need these two symptom targeting commits? 

https://github.com/lightningnetwork/lnd/pull/1349/commits/18de5587e679226a641226e94c08df9263ea150a
https://github.com/lightningnetwork/lnd/pull/1349/commits/418267eea9df206e58b0da607f0fef196dc40237

Reply:
Nah, still need those. This algo assumes that we are correctly closing only one connection